src/Idris/Desugar.idr

```diff
@@ -1022,7 +1022,9 @@ mutual
                                               map fst params) (mkNamespace recName))
                                fields
            let _ = the (List IField) fields'
-           let conname = maybe (mkConName tn) id conname_in
+           let conname = maybe (mkConName tn) snd conname_in
+           let condocs = maybe "" fst conname_in
```

Comment:
This is adding an empty docstring when there isn't any.
I'd rather we used `whenJust` and only add a docstring when there's actually one.

Reply:
Right, thanks! Same in the interface elab, fixed both.